<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1316</width>
    <height>1020</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа #8</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>951</width>
      <height>941</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="color_ind_edge">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QPushButton" name="but_color_edge">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выбрать
цвет
отсекателя</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_color_line">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выбрать
цвет
отрезков</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_color_edge_def">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>По
умолчанию</string>
    </property>
   </widget>
   <widget class="QLabel" name="color_ind_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QPushButton" name="but_color_line_def">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>По
умолчанию</string>
    </property>
   </widget>
   <widget class="QLabel" name="color_ind_clipped">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QPushButton" name="but_color_clipped">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выбрать
цвет
отсеченных</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_color_clipped_def">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>По
умолчанию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_close">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>760</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Замкнуть отсекатель</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_centre_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удерживайте CTRL, чтобы
добавить вертикальную линию</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lab_centre_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>710</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удерживайте SHIFT, чтобы
добавить горизонтальную линию</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="but_fill">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>810</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отсечь</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>860</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистить экран</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lab_centre_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Для добавления отрезка,
параллельного стороне отсекателя
выберите сторону в списке выше.
Затем нажмите кнопку ниже и укажите
середину требуемого отрезка</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lab_centre_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>900</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ЛКМ для добавления отрезков
ПКМ для добавления сторон отсекателя</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lab_centre_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Длина отрезка:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>590</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="but_close_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить параллельнный отрезок</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1316</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
